tests/benchdnn/doc/driver_matmul.md

```diff
@@ -21,7 +21,7 @@ where *matmul-knobs* are:
             specification for `src`, `weights`, and `dst` tensors through
             strides values. Refer to [option documentation](knob_strides.md)
             for details.
- - `--bia_dt={undef [default], f32, s32, s8, u8}` -- bias data type.
+ - `--bia-dt={undef [default], f32, s32, s8, u8}` -- bias data type.
```

Comment:
Is there any particular reason we aren't just fully spelling out this argument as `--bias-dt`? Shortening `bias` to `bia` just seems like it creates naming complexity for minimal benefit.

Reply:
The path of respecting the legacy... :)
But I'm fine to rename it in case we get some up votes.

Reply:
I believe we historically used fully spelled names in API (except for src/dst for some reason), and 3 letter shortening in benchdnn (src/dst/wei/bia). So unless we want to fully spell other names as well in benchdnn (e.g. `wei -> weights`), I would vote to keep it as is.